### PR TITLE
Fix duplicate ids in order page

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Make sure html are not duplicated.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Part in screenshots are working as expected.<br>image<br>image<br>image

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20626)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js [hunk 1]
```diff
@@ -105,6 +105,9 @@ export default class OrderProductEdit {
   displayProduct(product) {
     this.productRowEdit = $(OrderViewPageMap.productEditRowTemplate).clone(true);
     this.productRowEdit.attr('id', `editOrderProduct_${this.orderDetailId}`);
+    this.productRowEdit.find('*[id]').each(function removeAllIds() {
```
Comment (matks) [inline admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js:108]: 😆 
"A deleted ID is a good ID that does not mess with my code"

**Review by matks — DISMISSED** [review]
I approve the simple bugfix for 1.7.7 but I would like that we create an issue to fix this in a better way in 1.7.8 . 😉

**Comment by NeOMakinG** [comment]
> I approve the simple bugfix for 1.7.7 but I would like that we create an issue to fix this in a better way in 1.7.8 . 😉

Is there any better way ? I mean, removing every ids from form will break the whole BO, don't you think ? ^^

**Review by NeOMakinG — DISMISSED** [review]

File: admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js (hunk 1 above)
Comment (PierreRambaud) [reply admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js:108]: I want to die

**Comment by PierreRambaud** [comment]
> > I approve the simple bugfix for 1.7.7 but I would like that we create an issue to fix this in a better way in 1.7.8 . wink
> 
> Is there any better way ? I mean, removing every ids from form will break the whole BO, don't you think ? ^^

It removes ids from the template because it's not really a template :sweat_smile: 
For the 1.7.8.x it must be replaced by a CollectionType of forms and play with the prototype attribute.

**Review by NeOMakinG — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
It's WIP, I'm dumb 🤣

**Review by matthieu-rolland — DISMISSED** [review]
**LGTM**
blindbart

**Comment by Robin-Fischer-PS** [comment]
Hi @PierreRambaud !

Changing status from the Status block does not work with your PR. Changing it from the header of the order is OK.

pr20626 changing status KO

**Review by NeOMakinG — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Pull request merged**